app/models/twilio_redirection.rb

```diff
@@ -0,0 +1,31 @@
+class TwilioRedirection
+  def self.for(twilio_number)
+    if (location = ::Location.current.find_by(twilio_number: twilio_number))
```

Comment:
It raises a rubocop warning without the parenthesise. 

https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition